src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -84,6 +86,16 @@ public void SettingUpdatesProperties()
             SettingUpdatesPropertiesCore(item);
         }
 
+        [Fact]
+        public void SettingUpdatesPropertiesWhenReadOnly()
+        {
+            if (!CanBeReadOnly)
+                return; // directories can't be read only, so automatic pass
```

Comment:
Nit: we generally try to use braces for one-line bodies on different lines, especially when that's the style in the rest of the file